public/css/style.css

```diff
@@ -169,21 +184,179 @@ background: linear-gradient(rgba(96, 99, 100, 0.3), rgba(0,0,0,0.3));
   .play-foot, .login-btn {
     font: Bold 2rem 'Quicksand';
   }
-
-  #donations {
-    text-align: left;
+}
+.login-image-overlay {
+  width: 100%;
+  background:  linear-gradient(rgba(10, 10, 10, 0.6), rgba(10, 10, 10, 0.6)),
+      url('https://res.cloudinary.com/dqsmurjpg/image/upload/v1530770621/3-cards-against-humanity_32938.jpg');
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+.signin-image-overlay {
+  width: 100%;
+  background:  linear-gradient(rgba(10, 10, 10, 0.6), rgba(10, 10, 10, 0.6)),
+      url('https://res.cloudinary.com/dqsmurjpg/image/upload/v1530797290/cards_2.jpg');
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+@media screen and (min-width: 767px) {
+  #cfh-logo {
+    width: 25%
   }
+}
 
-  .donation-banner {
-    padding: 3rem;
+@media screen and (max-width: 767px) {
+  #cfh-logo {
+    width: 22%
   }
+}
 
-  .wish-logo {
-    top: 28rem;
+#auth {
+  height: 83vh;
+  text-align: center;
+}
+
+@media screen and (max-width: 600px) {
+  #auth {
+    height: 85vh;
   }
+}
 
-  .nav-wrapper {
-    margin-left: 2.5rem;
-    margin-right: 2.5rem;
+.auth-input {
+  width: 400px;
+  margin-left: auto;
+  margin-right: auto;
+}
+@media screen and (max-width: 420px) {
+  .auth-input {
+    width: 300px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+}
+@media screen and (min-width: 420px) {
+  .auth-input {
+    width: 400px;
+    margin-left: auto;
+    margin-right: auto;
   }
 }
+.input-field {
+  color:white;
+  box-sizing: border-box;
+  border: 2px solid #f1f1f1;
+  background-color: white;
+  border-radius: 3px;
+  margin-bottom: 0 !important;
```

Comment:
Avoid using `!important`. Try using [style specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) to help with applying styles to do what you want